app/routes/book_routes.py

```diff
@@ -49,6 +49,17 @@ def get_one_book(book_id):
         "description": book.description,
     }
 
+@books_bp.put("/<book_id>")
+def update_book(book_id):
+    book = validate_book(book_id)
+    request_body = request.get_json()
+
+    book.title = request_body.get("title")
+    book.description = request_body.get("description")
+    db.session.commit()
+
+    return Response(status=204)
```

Comment:
Should we include a comment that 204 is No Content? Learn will need to be updated to  talk about this as well, so not a lot of detail is needed in the code, but I think a reminder would be helpful.

Reply:
This technically returns a response with content type text/html. It's true there's no content, but I'd still prefer that our headers are consistent. We could set the content type explicitly, or we could stick more with our other returns and do something like
```suggestion
    return make_response({}, 204)
```

Reply:
My initial response if being a little offput by sending an empty dictionary instead of truly no content if that is the goal. I'm wondering if there's a reason `make_response({}, 204)` would be preferable to `Response(status=204, mimetype="application/json")`?

Reply:
Ha ha. Yeah. That's just enough to trigger the internal jsonify. I'm fine with adding the `mimetype` parameter, though.